dotnet/src/Functions/Functions.OpenApi/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -205,12 +205,13 @@ internal static List<RestApiOperation> CreateRestApiOperations(OpenApiDocument d
 
                 try
                 {
+                    var description = string.IsNullOrEmpty(operationItem.Description) ? operationItem.Summary : operationItem.Description;
                     var operation = new RestApiOperation(
                     id: operationItem.OperationId,
                     servers: operationServers,
                     path: path,
                     method: new HttpMethod(method),
-                    description: string.IsNullOrEmpty(operationItem.Description) ? operationItem.Summary : operationItem.Description,
+                    description: description?.Substring(0, Math.Min(description.Length, 1000)),
```

Comment:
Some scenarios may benefit from this change, while others may not. We shouldn't assume that this change works for all situations that SK addresses. So, if the CopilotAgentPlugin scenarios need it, consider applying it there.

Reply:
Could/should the value be an option somewhere without any default? If so where?

Reply:
Also beyond a configurable option, there probably should be a hard limit for service protection.

Reply:
The SK OpenAPI functionality provides customization capabilities that can be extended and used for cases like this - [OpenApiPlugin_Customization](https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/Plugins/OpenApiPlugin_Customization.cs).   
  
SK should tend to make as minimal assumptions about user scenarios as possible to remain maintainable and cater to a wide range of user scenarios, while simultaneously allowing for the tweaking of certain aspects for specific scenarios via customization.

Reply:
I guess my question is: what would be an acceptable API to have those sorts protections in semantic kernel (maybe off by default) which people could configure, without having to implement a decorator themselves?